cs/v1/README.md

```diff
@@ -1,8 +1,98 @@
+      - `Vessel Transport` and `Barge Transport` discriminator object changes
+        - Added `portVisitReference`,`transportCallReference`,`servicePartners`(object),`vessel`(object in vesselTransport),`barge`(object in bargeTransport)
+        - Added `MMSINumber`,`flag`,`callsign` to `vessel` and `barge` objects.
+        - Changed `vesselOperatorSMDGLinerCode` to `operatorCarrierCode` + `operatorCarrierCodeListProvider`
+        - Changed `vesselName` to `name`
+        - Changed `vesselIMONumber` length from String of 7 to String minimum length 7 and maximum length 8
+        - Changed maximum length of vessel name from 35 to 50
+        - Moved partner related properties `carrierCode`,`carrierCodeListProvider`,`carrierServiceName`,`carrierServiceCode`,`carrierImportVoyageNumber`,`carrierExportVoyageNumber` to the object(new) `servicePartners`
+    - location object changed from discriminators for each location to an object with types of different locations as optional properties
+    - Added `Next-Page-Cursor` to the response header for pagination
+    - Fixed typo in `errorCodeMessage` and is made mandatory
+    - Changed length of `vesselName` (now `name`) from 35 to 50
+    - Removed `name` from Address Location
+    - Changed `country` in address to `countryCode` and made mandatory.
+    - Added `minItems: 1` to the `Leg` array
+    - `sequenceNumber` format changed from int64 to int32
+    - `errorDateTime` example changed to UTC
+    - `dateTime` examples changed to date time with offset
+    - changed Facility SMDG Location to Facility Location and changed `facilitySMDGCode` to `facilityCode` and `facilityCodeListProvider` in `facility` in location object and added `BIC` to `facilityCodeListProvider`
+    - Changed inline `Detailed Error` to object
+- Port Schedules
+  - Filters
+    - Changed filter `port` to `UNLocationCode`
+    - Added `limit` and `cursor`
+  - Response
+    - location object changed from discriminators for each location to an object with types of different locations as optional properties
+    - Added `cutOffTimes`
+    - Added `servicePartners` object (mandatory) and moved properties `carrierCode`,`carrierCodeListProvider`,`carrierServiceName`,`carrierServiceCode`,`carrierImportVoyageNumber`,`carrierExportVoyageNumber` to it
+    - Added `vessel` object and moved `vesselIMONumber`,`name`(before `vesselName`),`operatorCarrierCode`,`operatorCarrierCodeListProvider` to it.
+    - Added `MMSINumber`,`flag`,`callsign` to `vessel` object
+    - Changed `vesselOperatorSMDGLinerCode` to `operatorCarrierCode` + `operatorCarrierCodeListProvider`
+    - Changed `vesselIMONumber` length from String of 7 to String minimum length 7 and maximum length 8
+    - Changed maximum length of vessel name from 35 to 50
+    - Changed Timestamp array attribute name from `timestamp` to `timestamps`
+    - Added `X-Next-Page-Cursor` to the response header for pagination
```

Comment:
Why is it called X-Next-Page-Cursor here when it is called Next-Page-Cursor in the endPoint described above?